ctl/src/command.rs

```diff
@@ -385,3 +385,76 @@ pub fn status(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>) {
     }
   }
 }
+
+pub fn remove_backend(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>, app_id: &str, ip: &str, port: u16) {
```

Comment:
there should be documentation for everything, but I don't feel comfortable requiring docs when I didn't write any in that project :p